### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by marysiakam9889**
added -ScanScheduleOffset
deprecation text for - ScanScheduleDay

### CODE REVIEW

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -991,14 +991,15 @@ Accept wildcard characters: False
 ```
 
 ### -ScanScheduleTime
-Specifies the time of day, as the number of minutes after midnight, to perform a scheduled scan.
-The time refers to the local time on the computer.
-If you do not specify a value for this parameter, a scheduled scan runs at a default time of two hours after midnight.
+This setting is discontinued and replaced with -ScanScheduleOffset as of January 2022 (version 4.18.2201.X) update to Microsoft Defender Antivirus
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:994]:
```suggestion
This setting is discontinued and replaced with -ScanScheduleOffset as of the January 2022 (version 4.18.2201.X) update to Microsoft Defender Antivirus.
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -991,14 +991,15 @@ Accept wildcard characters: False
...
+
+### -ScanScheduleOffset
+Configures the number of minutes after midnight to perform a scheduled scan. The time on the endpoint is used to determine the local time. If you enable this setting, a scheduled scan will run at the time specified. If you disable or do not enable this setting, a scheduled scan runs at the default time of two hours after midnight (120).
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:997]:
```suggestion
Configures the number of minutes after midnight to perform a scheduled scan. The time on the endpoint is used to determine the local time. If you enable this setting, a scheduled scan will run at the time specified. If you disable or do not enable this setting, a scheduled scan runs at the default time of two hours (120 minutes) after midnight.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
I am ok, I guess keeping ScanScheduleTime in the article is fine to give some time to people updating and still looking for it, however, I think the yaml portion also needs to be kept

**Review by dariomws — APPROVED** [review]
Agree with @get-itips.

**Review by tseward — APPROVED** [review]

**Comment by get-itips** [comment]
@marysiakam9889 Can you please review my comment? Thank you

**Review by denisebmsft — APPROVED** [review]
Looks good @marysiakam9889

**Pull request merged**